packages/framework/presence/src/presenceDatastoreManager.ts

```diff
@@ -422,15 +437,66 @@ export class PresenceDatastoreManagerImpl implements PresenceDatastoreManager {
 	 * correlation with other telemetry where it is often called just `clientId`.
 	 */
 	private prepareJoinResponse(
-		updateProviders: ClientConnectionId[],
 		requestor: ClientConnectionId,
+		updateProviders?: ClientConnectionId[],
```

Comment:
Can be undone with the restore to message protocol.
But it is worth noting that no caller skip providing this argument. A more accurate typing would be `updateProviders: ClientConnectionId[] | undefined`.